src/XRP/protobuf-wrappers/index.ts

```diff
@@ -9,7 +9,7 @@ export { default as XrpDepositPreauth } from './xrp-deposit-preauth'
 export { default as XrpEscrowCancel } from './xrp-escrow-cancel'
 export { default as XrpEscrowCreate } from './xrp-escrow-create'
 export { default as XrpEscrowFinish } from './xrp-escrow-finish'
-export { default as XrpIssuedCurrency } from './xrp-issued-currency'
+export { default as XrpIssuedCurrency } from './xrpl-issued-currency'
```

Comment:
and then I think we want to make a note of this name change in the CHANGELOG?  since I think this is technically a breaking change... though I'm not sure `XrpIssuedCurrency` was exported at the top level anyway.  I don't think we need to do any deprecations because this class was unusable anyway